source/pervasive/nonlinear_arith/div.rs

```diff
@@ -0,0 +1,1088 @@
+//! Lemmas for division.
+
+use builtin::*;
+use builtin_macros::*;
+#[allow(unused_imports)]
+use crate::calc_macro::*;
+
+verus! {
+
+#[allow(unused_imports)]
+use crate::nonlinear_arith::internals::div_internals::{div_recursive, lemma_div_induction_auto, div_auto, div_pos, lemma_div_auto};
+#[allow(unused_imports)]
+use crate::nonlinear_arith::internals::div_internals_nonlinear as DivINL;
+#[allow(unused_imports)]
+use crate::nonlinear_arith::internals::mod_internals::{lemma_mod_auto, mod_recursive};
+#[allow(unused_imports)]
+use crate::nonlinear_arith::internals::mod_internals_nonlinear as ModINL;
+#[allow(unused_imports)]
+use crate::nonlinear_arith::internals::mul_internals::{lemma_mul_induction_auto, lemma_mul_auto, mul_auto, lemma_mul_induction};
+#[allow(unused_imports)]
+use crate::nonlinear_arith::mul::*;
+#[allow(unused_imports)]
+use crate::nonlinear_arith::internals::general_internals::{is_le};
+#[allow(unused_imports)]
+use crate::nonlinear_arith::modulus::*;
```

Comment:
Are these actually unused imports at this point? If they are not unused imports, the `allow` should be removed for them; if they _are_ unused, then the `use` declaration should be removed.

Also do check all the other files for similar `unused_imports` declarations.